src/openrct2/actions/FootpathPlaceAction.hpp

```diff
@@ -76,12 +76,12 @@ DEFINE_GAME_ACTION(FootpathPlaceAction, GAME_COMMAND_PLACE_PATH, GameActionResul
             return MakeResult(GA_ERROR::DISALLOWED, STR_CANT_BUILD_FOOTPATH_HERE, STR_LAND_SLOPE_UNSUITABLE);
         }
 
-        if (_loc.z / 8 < FootpathMinHeight)
+        if (_loc.z < FootpathMinHeight)
```

Comment:
Isn't this a potentially different result?

Reply:
Doubt it, as it won't let you build at z coordinates that aren't multiples of 1.5 metres anyway.

Reply:
Alright